dask_cloudprovider/azure/azurevm.py

```diff
@@ -241,6 +241,19 @@ class AzureVMWorker(WorkerMixin, AzureVM):
     """Worker running on an AzureVM."""
 
 
+class _Config(dict):
+    """Simple config interface for `AzureVMCluster`.
+
+    Enables '.' notation for nested access, as per `dask.config.get`.
+
+    """
+    def __new__(cls, d):
+        return super().__new__(cls, d)
+        
+    def get(self, key, default=None, override_with=None):
+        return dask.config.get(key, default=default, config=self, override_with=override_with)
+
+
```

Comment:
This would be useful in a bunch of places, would you mind moving it somewhere like `dask_cloudprovider.utils` or maybe even `dask_cloudprovider.config`?

Also happy for it to be a public class.

Reply:
Sure thing. Do you also want more detailed documentation for the class?

Reply:
Perhaps just a small `Example` block would be helpful for future implementers (probably me).